src/main/java/net/glowstone/entity/GlowAnimal.java

```diff
@@ -44,6 +44,30 @@ protected int getAmbientDelay() {
         return 120;
     }
 
+    /**
+     * Determines whether this entity can eat an item, if so, applies the effects of eating it.
+     *
+     * @param player the player feeding the entity, for statistical purposes
+     * @param type an item that may be food
+     * @return true if the item should be consumed; false otherwise
+     */
+    protected boolean tryFeed(Material type, GlowPlayer player) {
+        if (getBreedingFoods().contains(type)) {
```

Comment:
Remove a nesting level by inverting this if-statement.

Reply:
Done.